src/web-ui/.env

```diff
@@ -41,4 +41,5 @@ VUE_APP_PINPOINT_REGION=us-west-2
 VUE_APP_SEGMENT_WRITE_KEY=
 
 # Configure Moovweb XDN
-VUE_APP_XDN_ENABLED=false
\ No newline at end of file
+VUE_APP_XDN_ENABLED=false
+XDN_DEPLOY_TOKEN=
```

Comment:
@james-jory here I defined `XDN_DEPLOY_TOKEN` without `VUE_APP_` prefix part because it must not be exposed on the client side js (`dist` folder build)

Reply:
Okay, please just add a comment to this file too explaining the reason since it's different than the others and someone could think it's named incorrectly in the future and change it.